heuristic/classes/Route.py

```diff
@@ -56,3 +56,72 @@ def _plan_cost(self, problem: Problem) -> float:
             cost += Stacks.cost(customer, problem, before, after)
 
         return cost
+
+    def is_feasible(self, problem: Problem, customer: int) -> bool:
+        """
+        Checks if inserting a customer in this route is feasible.
+        """
+        delivery = Item(problem.demands[customer], DEPOT, customer)
+        pickup = Item(problem.pickups[customer], customer, DEPOT)
+
+        if all(stacks.shortest_stack().volume() + delivery.volume <=
```

Comment:
Difficult to read. Split it into can_deliver and can_pickup, two boolean variables, and return `can_deliver and can_pickup`.